spec/rspec/core/reporter_spec.rb

```diff
@@ -36,6 +35,12 @@ module RSpec::Core
         reporter.finish
       end
 
+      it "will allow profiler to be used without being properly setup" do
+        allow(config).to receive(:profile_examples?) { true }
```

Comment:
Does this have to stub `config`?  Or can it repro the issue using `config.profile_examples = true`?  In general, I prefer avoiding stubbing in these situations unless it's necessary.

Reply:
No it doesn't I changed it because of your earlier concern that this was activating the formatter.